### ISSUE TITLE
Spring cloud bindings 1.8.0 release got overwritten

### ISSUE THREAD

**Issue by petermylemans** [spring-cloud/spring-cloud-bindings#61]
I'm building containers using Spring Boot's buildpack integration and the paketo builder (the default one) that downloads spring cloud bindings 1.8.0 and does a checksum on the downloaded release artefact.

This stopped working just now due to the jar of a release 1.8.0 being replaced and thus receiving a new checksum:

```
 sha256 for spring-cloud-bindings-1.8.0.jar 38657efbee82aaf69ea4998d5839b701173b8ccd01db0f97c3a16f86b2436bf7 does not match expected c75ef3152817c28c7049d7e0052dfba3ed461a7151984e594b656c0be927888e
```

A quick look a the repo seems to suggest the same thing:  https://repo.spring.io/ui/native/release/org/springframework/cloud/spring-cloud-bindings/1.8.0

1.8.0. was modified at 07-01-22 17:10:03 +0100 (around the same time as 1.8.1 release).

Validating the sha256 checksum for external released dependencies is quite a common use case, so I would expect it to remain stable once the release was made.

Did something go wrong in the release process?

**Comment by focus-on-it** [comment]
We arrived at exactly the same conlusion a couple of minutes ago

**Comment by dmikusa-pivotal** [comment]
Yes, apologizes. Working on fixing this now. Sorry.

**Comment by petermylemans** [comment]
We've all been there and pushed the wrong button ;) Thanks for the feedback and hope you get it working soon!

**Comment by dmikusa-pivotal** [comment]
So there's a new paketo-buildpacks/java release that fixes this, 6.5.1. It should get picked up by the builders in 15-30 minutes. In the meantime, you can use `gcr.io/paketo-buildpacks` as the `-b` argument to `pack build` or as a `<buildpack>` entry in the Spring Boot maven configuration. That'll point directly to the buildpack, which should get the latest version. You can remove this extra config once the builders update.

**Comment by dmikusa-pivotal** [comment]
We have also managed to restore the 1.8.0 JAR file, so if you're on an older version or builder that should be working again. Sorry again for the inconvenience.
